pkg/skaffold/debug/transform_go.go

```diff
@@ -0,0 +1,201 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package debug
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/util"
+
+	"github.com/sirupsen/logrus"
+	v1 "k8s.io/api/core/v1"
+)
+
+type dlvTransformer struct{}
+
+func init() {
+	containerTransforms = append(containerTransforms, dlvTransformer{})
+}
+
+const (
+	// dlv defaults to port 56268
+	defaultDlvPort    = 56268
+	defaultAPIVersion = 2
+)
+
+// dlvSpec captures the useful nodejs devtools options
```

Comment:
useful *go* devtools options? Or is nodejs involved here somehow?

Reply:
oops.